src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties

```diff
@@ -293,6 +293,7 @@ RSC_019=EPUBs with Multiple Renditions should contain a META-INF/metadata.xml fi
 RSC_020='%1$s' is not a valid URI.
 RSC_021=A Search Key Map Document must point to Content Documents ('%1$s' was not found in the spine).
 RSC_022=Cannot check image details (requires Java version 7 or higher).
+RSC_023='%1$s' is not a valid URL.
```

Comment:
I agree, not sure what's the most understandable option... (my useless 2 cents ;-)